pydexec/command.py

```diff
@@ -18,6 +18,65 @@ def args(self, *args):
         self._args.extend(args)
         return self
 
+    def env(self, env_key, env_val):
+        """
+        Add the environment variable ``env_key`` with the value ``env_val``.
+        """
+        self._env[env_key] = env_val
+        return self
+
+    def env_remove(self, env_key):
+        """ Remove the environment variable ``env_key``. """
+        del self._env[env_key]
+        return self
+
+    def env_clear(self):
+        """ Clear all environment variables. """
+        self._env = {}
+        return self
+
+    def env_arg(self, env_key, default=None, required=False, remove=True):
+        """
+        Add an argument with the value of the ``env_key`` environment variable
+        if it is set.
+
+        :param env_key: environment variable name to use
+        :param default: default value if the environment variable is not set
+        :param required: raise an error if the environment variable is not set
+        :param remove: remove the variable from the command's environment
+        """
```

Comment:
Do we want `remove=True` as the default? I'd expect the envvar to remain set unless I explicitly asked for it to be removed.